modules/wmco-disconnected-cluster.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// windows_containers/enabling-windows-container-workloads.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="wmco-disconnected-cluster_{context}"]
+= Using Windows containers with a mirror registry
+
+The Windows Machine Config Operator (WMCO) can pull images from a registry mirror rather than from a public registry by using an `ImageDigestMirrorSet` (IDMS) or `ImageTagMirrorSet` (ITMS) object to configure your cluster to pull images from the mirror registry.
+
+A mirror registry has the following benefits: 
+
+* Avoids public registry outages 
+* Speeds up node and pod creation
+* Pulls images from behind your organization's firewall
+
+A mirror registry can also be used with a {product-title} cluster in a disconnected, or air-gapped, network. A _disconnected network_ is a restricted network without direct internet connectivity. Because the cluster does not have access to the internet, any external container images cannot be referenced.
+
+Using a mirror registry requires the following general steps:
+
+* Create the mirror registry, using a tool such as Red Hat Quay.
+* Create a container image registry credentials file.
+* Copy the images from your online image repository to your mirror registry.
+
+For information on these steps, see "About disconnected installation mirroring."
```

Comment:
Rewrite as: "For information about these steps..."

Reply:
Agree with above